### ISSUE TITLE
Attributes for s_user not created by API resource

### ISSUE THREAD

**Issue by samuelvogel** [VIISON/shopware-4#25]
When creating a customer through the API resource, not entry in `s_user_attributes` is created with Shopware `5.2.16`.

Example:
`curl -i 'http://localhost/shopware/api/customers' --DIGEST -u user:key --globoff -H 'content-type: application/json' -d '{"email":"test@example.com","salutation":"mr","firstname":"Max","lastname":"Tester","billing":{"salutation":"mr","firstname":"Max","lastname":"Tester","street":"Musterstr. 1","zipcode":"12345","city":"Musterstadt","country":2}}'`

**Comment by samuelvogel** [comment]
This especially affects customer accounts created via Pickware POS.

**Comment by fixpunkt** [comment]
Is this an error? From my understanding, attribute entities are only created when an attribute value is defined for the entity.

**Comment by samuelvogel** [comment]
Yes, the corresponding attributes should always be there. When registering via the frontend and manually creating a customer via the backend the attributes exist as well.

**Comment by fixpunkt** [comment]
That's because these UIs create an attribute entity. This is not generally true for entity attributes.
